File: common/Models/VisitInfo.cs
using System;
namespace Common.Models
{
    public class VisitInfo
    {
        public string id;

        public string PartitionKey;

        // Links VisitInfo and VisitorInfo
        public string VisitorInfoId;

        public string Organization;

        public string DateTime;

        public string Date;

        public string Time;

        public string Door;

        public string Direction;

        public int? EventId;

        public bool? BookingOverride;

        public bool? CapacityOverride;

        public bool? Offline;

        public string Gender;

        public string ScannerVersion;

        public string DeviceId;

        public string DeviceLocation;

        public string DocType = "Visit";
    }
}

File: Scanner/Android/app/src/main/java/ca/snmc/scanner/models/VisitInfo.kt
package ca.snmc.scanner.models

import java.util.*

data class VisitInfo(
    var visitorId: UUID?,
    var organization: String?,
    var door: String?,
    var direction: String?,
    var eventId: Int?,
    var bookingOverride: Boolean?,
    var capacityOverride: Boolean?,
    var scannerVersion: String?,
    var deviceId: String?,
    var deviceLocation: String?,
    var dateTimeFromScanner: String?,
    var anti_duplication_timestamp: Long?
)